libpod/container_api.go

```diff
@@ -341,6 +341,10 @@ func (c *Container) Exec(tty, privileged bool, env map[string]string, cmd []stri
 			logrus.Errorf(lastErr.Error())
 		}
 		lastErr = errors.Wrapf(define.ErrOCIRuntime, "non zero exit code: %d", exitCodeData.data)
+		// if we failed in podman, set the exit code to a generic error
```

Comment:
"if we failed in **podman**"?

Reply:
I updated the comment to be more informative